BlogAPI.Persistence/Migrations/BlogDbContextModelSnapshot.cs

```diff
@@ -0,0 +1,120 @@
+﻿// <auto-generated />
+using System;
+using BlogAPI.Persistence;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+#nullable disable
+
+namespace BlogAPI.Persistence.Migrations
+{
+    [DbContext(typeof(BlogDbContext))]
+    partial class BlogDbContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
```

Comment:
🤨

Reply:
Generated by EF Core when running add-migration, don't think it's too important here